scripts/workers/code_upload_submission_worker.py

```diff
@@ -10,8 +10,8 @@
 
 # TODO: Add exception in all the commands
 from kubernetes.client.rest import ApiException
-from statsd_utils import increment_and_push_metrics_to_statsd
-from worker_utils import EvalAI_Interface
+from .statsd_utils import increment_and_push_metrics_to_statsd
+from .worker_utils import EvalAI_Interface
```

Comment:
Why was this change needed?

Reply:
@AyushR1 Are you sure this doesn't break any of the existing code? For example, if I create a code-upload challenge, the worker will work fine?